composer.json

```diff
@@ -1,5 +1,6 @@
 {
     "require": {
-        "fzaninotto/faker": "dev-master"
+        "fzaninotto/faker": "dev-master",
+        "symfony/process": "^5.4.0"
```

Comment:
@bx80 Why are we not adding the vendor folder changes with this PR ?

Reply:
I wasn't sure what the approach was for plugin specific composer dependencies, I've now added the updated vendor files.